.gitlab/internal_kubernetes_deploy.yml

```diff
@@ -25,7 +25,7 @@ internal_kubernetes_deploy_experimental:
     OPTION_PRE_SCRIPT: "patch-cluster-images-operator.sh snowver ${CI_COMMIT_REF_SLUG}-jmx-${CI_COMMIT_SHORT_SHA} ${CI_COMMIT_REF_SLUG}-${CI_COMMIT_SHORT_SHA}"
     SKIP_PLAN_CHECK: "true"
     WORKFLOW: "agents"
-    FILTER: "cluster.env == 'experimental' and cluster.shortName == 'snowver'"
+    FILTER: "cluster.shortName in ['stripe', 'snowver']"
```

Comment:
I think that, unless you update line 25 as well, this will just redeploy whatever is in k8s-datadog-agent-ops, instead of deploying the nightly.

It looks like line 25 runs [this script](https://github.com/DataDog/k8s-datadog-agent-ops/blob/af8329e5b712b3b029851679a798577082812ed9/scripts/patch-cluster-images-operator.sh#L1), and it looks like this script will probably need to be updated in some capacity in order to work with this cluster

Reply:
good point I'll update this